app/src/main/java/com/hieuminh/chatapp/presenter/LoginPresenter.kt

```diff
@@ -0,0 +1,20 @@
+package com.hieuminh.chatapp.presenter
+
+interface LoginPresenter {
```

Comment:
extract this fun to Contract class. in Contract class have 2 interface are view and presenter